protocol/flatbuffers/settings/model.fbs

```diff
@@ -0,0 +1,29 @@
+namespace solarxr_protocol.rpc.settings;
+
+/// Settings for the skeletal model that are toggles.
+table ModelToggles{
+    extended_spine: bool;
+    extended_pelvis: bool;
+    extended_knee: bool;
+    force_arms_from_hmd: bool;
+}
+
+/// Settings for the skeletal model that are ratios.
+/// These values range from 0 to 1.
+table ModelRatios{
+    waist_spine_averaging: float32;
+    waist_pelvis_averaging: float32;
+
+    // TODO, rename to be more clear
+    hip_pelvis_averaging: float32;
```

Comment:
We should name this clearly, before merging

Reply:
`waist_spine_averaging` is the waistNode interpolating between the chest and hip when the waist tracker is missing.
`waist_pelvis_averaging` is the waist interpolating between the chest and pelvis* when the waist tracker is missing.
`hip_pelvis_averaging` is the hip interpolating between a spine tracker (waist or chest) and the pelvis* when the hip tracker is missing.

`pelvis_hip_averaging` is the hipNode (not tracker) interpolating between itself and the pelvis*
`waist_tracker_pelvis_averaging` is the trackerWaistNode interpolating between itself and the pelvis on its yaw and roll only.

`knee_tracker_angle_averaging` is the kneeTrackerNodes interpolating between themselves and the ankles on their yaw and roll only.

*Pelvis = average of the two upper leg trackers.